RESULTS.md

```diff
@@ -0,0 +1,91 @@
+
+The colours of the journey map blocks signify if it is a screen that users will interact with (orange) or if it just a confirmation/informational screen (blue). As well the icons in the top right of the screens themselves indicate whether or not that screen was implemented sucessfully (green checkmark), In Progress (grey clock icon) or unsuccessful implementation (red "X"). 
+If there is a red "X" you will find the reason for the unsuccessful implementation in the following "Findings" section.
+
+The Journey Map blocks also describe what the user should expect depending on which "mode" the application is. The two available modes are "Conversation mode ON" which indicates that the user is speaking using the microphone to navigate the application (the screen on the left represents the layout for this mode), or "Conversation Mode OFF" which uses the buttons presented on the screen to navigate the application (the screen on the right represents the layout for this mode).
+
+The final component of a Journey Map block is the "Technical Considerations" section, this outlines the technical considerations and if they were achieved or not. Like with the above screens these too have an icon depicting if the implementation of the technical components were successful or unsuccesful. 
+
+
+## Findings
+
+### WebXR
+1. WebXR is currently a Work in Progress standard, but working towards standardization. Due to this the support for WebXR in Operating Systems and mainstream browsers is limited. 
+2. Full support of the features in the prototype was only available on Android devices with ARCore and with Chrome version 87+. 
+3. Limited support of the features in the prototype were offerred to iOS users using the WebXR Viewer application released by Mozilla. As well as for development Mozilla released an extension that could be installed in Chrome and Firefox to simulate the WebXR environment. However both of these applications were limited in that they could not interact with the BabylonJS GUI elements.
+ 
+### BabylonJS
+1. Due to Security policies in Chrome, the AudioContext is not allowed to start until user input/interaction is done with the webpage. This meant that the prototype could not speak an initial welcome message, and was remedied by manually unlocking the AudioContext engine before the welcome message was played.
+2. GUI Text Input does not function properly on a mobile. This is a currently known issue within the BabylonJS documentation, the Text Input element cannot invoke the onscreen keyboard so as a workaround it sends an alert window to get the input. This however causes the AR world to freeze as on mobile this prompt window does not get sent to the front above the AR world. 
```

Comment:
GUI Text input does not function properly on mobile.

Reply:
This however causes the AR world to freeze on a mobile device as the window prompt does not retain focus while in the AR world. (not sure if this accurate sentence)

Reply:
Updated the document with the suggested edits.